src/Database.hpp

```diff
@@ -1,31 +1,91 @@
 #pragma once
 #include "CNStructs.hpp"
 #include "Player.hpp"
-#include <list>
 #include <string>
+#include <vector>
 
 namespace Database {
+#pragma region DatabaseStructs
+
+	struct Account
```

Comment:
Tab indentation here as well, let's fix that

Reply:
fixed, and changed my editor settings for the future